tests/ci_install.sh

```diff
@@ -40,12 +42,12 @@ fi
 if [[ ${dependencies} == "minimal" ]]; then
     # test with minimal versions of runtime dependencies
     python -m pip install -e ".[test,minimal]"
+elif [[ ${dependencies} == "cartopy" ]]; then
+    python -m pip install 'cartopy>=0.22'
```

Comment:
So I think if we add in a pip install scipy here as well then we can remove the two scipy install lines above (on lines 35 and 15).

Reply:
I **do** think we still need an explicit scipy install since it's an optional cartopy dep and an optional yt dep (but we're using the cartopy functionality for which it is required). It is in our full deps though, so we don't need it down below.